tests/test_log.py

```diff
@@ -44,6 +44,10 @@ def test_log_constructor():
     pageUrl = test_log.data.page_url
     assert pageUrl == "https://github.com/apache/flagon/tree/master/docker"
 
+    id = test_log.id
+    assert id.get_timestamp() == 1719530111079 // 1000
```

Comment:
As long as this passes it is fine, but I do not use floor division when parsing the timestamp. Just a nit pick though